Containerfile

```diff
@@ -5,10 +5,8 @@ RUN dnf update -y && dnf clean all
 
 # install dependencies in a separate layer to save dev time
 WORKDIR /app
-COPY package.json package-lock.json . 
-RUN npm install
-
 COPY . .
+RUN npm install
```

Comment:
npm install was failing without the rest of the project?
asking because having the installation of dependencies in a dedicated layer usually benefits the developer with faster local builds (coming from the premise dependencies don't change as often as the rest of the code).

Reply:
nevermind, in a subsequent commit I see the installation of dependencies were added back (https://github.com/quipucords/quipucords-ui/pull/342#discussion_r1571387121). 

With that, is this line still necessary?